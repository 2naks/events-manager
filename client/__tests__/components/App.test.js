import ConnectedApp, { App } from '../../containers/App';

describe('App Component', () => {
  const initialState = {
  };

  const middlewares = [thunk];
  const mockStore = configureStore(middlewares);
  const store = mockStore(initialState);

  describe('App form  should render self', () => {
    it('it should render for dumb component', () => {
      const wrapper = shallow(<App />);
      expect(wrapper).toMatchSnapshot();
    });

    it('it should render connected component', () => {
      const wrapper = shallow(<Provider store={store}><ConnectedApp /></Provider>);
      expect(wrapper.length).toEqual(1);
    });
  });
});
