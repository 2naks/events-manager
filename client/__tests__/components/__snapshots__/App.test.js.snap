// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component App form  should render self it should render for dumb component 1`] = `<LandingPage />`;
